src/Checkbox.jsx

```diff
@@ -96,23 +93,17 @@ export default class Checkbox extends React.Component {
     const state = this.state;
     const prefixCls = props.prefixCls;
     let checked = state.checked;
-    if (typeof checked === 'boolean') {
-      checked = checked ? 1 : 0;
-    }
-    const className = classNames({
-      [props.className]: !!props.className,
-      [prefixCls]: 1,
+
+    const className = classNames([prefixCls], {
+      [props.className]: props.className,
```

Comment:
Because `classNames` library will auto convert value to boolean,so we don't need use `!!`